tests/testthat/setup.R

```diff
@@ -1,35 +1,46 @@
 on_cran <- function() {
   !interactive() && !isTRUE(as.logical(Sys.getenv("NOT_CRAN", "false")))
 }
+
 if (!on_cran()) {
   withr::local_envvar(
     R_USER_CACHE_DIR = tempfile(),
     .local_envir = testthat::teardown_env(),
-    EUNOMIA_DATA_FOLDER = Sys.getenv("EUNOMIA_DATA_FOLDER", unset = tempfile()),
-    DB_TO_TEST = "duckdb" #"postgres" # Write which db you want to use
+    EUNOMIA_DATA_FOLDER = Sys.getenv("EUNOMIA_DATA_FOLDER", unset = tempfile())
   )
   CDMConnector::downloadEunomiaData(overwrite = TRUE)
 }
+
 connection <- function(type = Sys.getenv("DB_TO_TEST", "duckdb")) {
   switch(
     type,
-    "duckdb" = DBI::dbConnect(duckdb::duckdb(), ":memory:")
+    "duckdb" = DBI::dbConnect(duckdb::duckdb(), ":memory:"),
+    "postgres" = DBI::dbConnect(RPostgres::Postgres(),
+                                dbname = Sys.getenv("server_dbi"),
+                                port = Sys.getenv("port"),
+                                host = Sys.getenv("host"),
+                                user = Sys.getenv("user"),
+                                password = Sys.getenv("password"))
   )
 }
+
 schema <- function(type = Sys.getenv("DB_TO_TEST", "duckdb")) {
   switch(
     type,
-    "duckdb" = c(schema = "main", prefix = "os_")
+    "duckdb" = c(schema = "main", prefix = "os_"),
+    "postgres" = c(schema = "results", prefix = "os_")
   )
 }
+
 cdmEunomia <- function() {
   con <- connection()
   schema <- schema()
   conDuck <- DBI::dbConnect(duckdb::duckdb(), CDMConnector::eunomia_dir())
   cdmDuck <- CDMConnector::cdmFromCon(
     con = conDuck, cdmSchema = "main", writeSchema = "main"
   )
-  cdm <- CDMConnector::copyCdmTo(con = con, cdm = cdmDuck, schema = schema)
+  cdm <- CDMConnector::copyCdmTo(con = con, cdm = cdmDuck, schema = schema, overwrite = TRUE)
```

Comment:
if you need overwrite = TRUE is that you are not cleaning previous tables. I can accept PR, but we should try to fix this @martaalcalde